improver/utilities/cube_manipulation.py

```diff
@@ -259,45 +282,30 @@ def process(
         result of premature iris merging on load).
 
         Args:
-            cubes_in:
+            *cubes:
                 Cubes to be merged.
-            check_time_bounds_ranges:
-                Flag to check whether scalar time bounds ranges match.
-                This is for when we are expecting to create a new "time" axis
-                through merging for eg precipitation accumulations, where we
-                want to make sure that the bounds match so that we are not eg
-                combining 1 hour with 3 hour accumulations.
-            slice_over_realization:
-                Options to combine cubes with different realization dimensions.
-                These cannot always be concatenated directly as this can create a
-                non-monotonic realization coordinate.
-            copy:
-                If True, this will copy the cubes, thus not having any impact on
-                the original objects.
 
         Returns:
             Merged cube.
         """
-        # if input is already a single cube, return unchanged
-        if isinstance(cubes_in, iris.cube.Cube):
-            return cubes_in
```

Comment:
Has this been removed because it will be captured by the following `if len(cubes) == 1:` section? I think this is right, but just checking.